sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegenExec.scala

```diff
@@ -709,7 +709,12 @@ case class WholeStageCodegenExec(child: SparkPlan)(val codegenStageId: Int)
     val duration = System.nanoTime() - startTime
     WholeStageCodegenExec.increaseCodeGenTime(duration)
 
-    logDebug(s"\n${CodeFormatter.format(cleanedSource)}")
+    val formatCleanCode = CodeFormatter.format(cleanedSource)
+    if (conf.wholeStageCleanCodePrintEnabled) {
+      logInfo(s"\n$formatCleanCode")
```

Comment:
I don't think it's worth introducing another PR. I believe you can already configure this via log4j configurations

Reply:
Thanks for reviewing. If I turn log level to DEBUG, there are logs more than I need. So I try to configure this on the other way

Reply:
+1 Agree with @HyukjinKwon 

@TonyDoen  you can just configure `org.apache.spark.sql.execution.WholeStageCodegenExec` to debug